Extensions/accesskit.js

```diff
@@ -108,9 +108,69 @@ XKit.extensions.accesskit = new Object({
 		}
```

Comment:
Would be nice to have a thing on `contrast` above saying it's deprecated or something

Reply:
i was thinking of hiding the deprecated options on the react site, so ill give that a proper try

Reply:
Neato burrito